docs/custom-audio.md

```diff
@@ -0,0 +1,15 @@
+# Custom Audio
+To replace a sequences you should only create a zip file with textures to replace at the same path as mk64.o2r. Only wav, mp3, ogg and flac are supported.
+## Example:
+You want to change sound/sequences/main_menu then you just need to create a zip file with the following structure:
```

Comment:
The following zip structure is required to replace sounds for `sound`, `sequences`, and `main_menu`. Use the same name as the old sound file name but with your extension instead.

Reply:
I keep the same structure then https://github.com/HarbourMasters/SpaghettiKart/blob/main/docs/textures-pack.md#example so maybe make a pr to replace both should be better.